DB/WorkWithUsers/Create.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LanitHW4.DB.WorkWithUsers
+{
+    public class Create : Read
+    {
+        public void CreateGame() 
+        {
+            ReadGames();
+            DbGames games = new DbGames();
+
+            Console.Write("Введите следущий game_id: ");
+            games.Game_id = int.Parse(Console.ReadLine());
+
+            Console.Write("Введите название команды: ");
+            games.Team = Console.ReadLine();
+
+            Console.Write("Введите город: ");
+            games.City = Console.ReadLine();
+
+            Console.Write("Введите количетво голов: ");
+            games.Goals = int.Parse(Console.ReadLine());
+
+            Console.Write("Введите время когда был сыгран матч в формате YYYY-MM-DD HH:MM: ");
+            games.Game_data = DateTime.Parse(Console.ReadLine());
+
+            using(FootboldbContext context = new FootboldbContext())
+            {
+                context.Games.Add(games);
+                context.SaveChanges();
+
+                DbGames games1 = context.Games.LastOrDefault();
```

Comment:
there is PK in table -> position of new record would be chosen according PK order; so it can be last, first or middle
to show record can use
`DbGames games1 = context.Games.FirstOrDefault((g => g.Id == games.Id);`